### PR TITLE
Integration tests for mac_ports.py

### PR DESCRIPTION
**Pull request by twangboy**
### What does this PR do?

Adds integration tests for mac_ports.py
### What issues does this PR fix or reference?

https://github.com/saltstack/zh/issues/553
### Tests written?

Yes

### CODE REVIEW

File: tests/integration/modules/mac_ports.py [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+
+        self.AGREE_INSTALLED = 'agree' in self.run_function('pkg.list_pkgs')
+        self.run_function('pkg.refresh_db')
+
+    def tearDown(self):
+        '''
+        Reset to original settings
+        '''
+        if not self.AGREE_INSTALLED:
+            self.run_function('pkg.remove', ['agree'])
```
Comment (cachedout) [inline tests/integration/modules/mac_ports.py]: Does this remove a package if it isn't installed?

Comment (cachedout) [inline tests/integration/modules/mac_ports.py]: Also, this won't restore us to original settings. If a person had this on their system and needed it, they'll be left without it.

File: tests/integration/modules/mac_ports.py [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+    def test_latest_version(self):
+        '''
+        Test pkg.latest_version
+        '''
+        self.run_function('pkg.install', ['agree'])
+        result = self.run_function('pkg.latest_version',
+                                   ['agree'],
+                                   refresh=False)
+        self.assertIsInstance(result, dict)
+        self.assertIn('agree', result)
```
Comment (cachedout) [inline tests/integration/modules/mac_ports.py]: How does this test that this is the latest version? This looks like it just tests that it's installed.

File: tests/integration/modules/mac_ports.py (hunk 1 above)
Comment (twangboy) [inline tests/integration/modules/mac_ports.py]: In the setUp function it determines if `agree` is installed or not and sets that to `self.AGREE_INSTALLED`. True if installed, False if not.
In tear down, if it wasn't installed before, it uninstalls it to leave the machine in the same state.

File: tests/integration/modules/mac_ports.py (hunk 2 above)
Comment (twangboy) [inline tests/integration/modules/mac_ports.py]: There's no way of knowing when the test is run what the latest version of agree will be. It's currently at 3.0. All I can do is check that it returned a dict containing 'agree'.

**Comment by twangboy** [comment]
Rebased

**Pull request merged**